cleverhans/attacks.py

```diff
@@ -25,16 +363,17 @@ def fgsm(x, predictions, eps, back='tf', clip_min=None, clip_max=None):
                     value for components of the example returned
     :return: a tensor for the adversarial example
     """
+    warnings.warn("attacks.fgsm is deprecated and will be removed on 09-27-17."
```

Comment:
In that case, would it make more sense to create a FastGradientSignMethod class to be consistent (and then just have it use FastGradientMethod but with ord=np.inf)?